css/css-pseudo/META.yml

```
@@ -171,3 +171,26 @@ links:
       url: https://bugzilla.mozilla.org/show_bug.cgi?id=1675231
       results:
         - test: first-line-with-inline-block.html
+    - product: chrome
+      url: https://crbug.com/490511
+      results:
+        - test: first-letter-001.html
+        - test: idlharness.html
+    - product: chrome
+      url: https://crbug.com/1172333
+      results:
+        - test: first-letter-digraph.html
+    - product: chrome
+      url: https://crbug.com/1163175
+      results:
+        - test: first-letter-punctuation-and-space.html
+    - product: chrome
+      url: https://crbug.com/1147859
+      results:
+        - test: highlight-painting-order.html
```

```suggestion
        - test: css/css-pseudo/highlight-painting-001.html
```

It was renamed in https://bugs.chromium.org/p/chromium/issues/detail?id=1147859#c20 and the same bug still seems relevant. https://wpt.fyi/results/css/css-pseudo/highlight-painting-001.html?label=master&label=experimental&aligned fails.